api/src/main/java/jakarta/enterprise/inject/build/compatible/spi/AnnotationBuilder.java

```diff
@@ -16,6 +16,9 @@
  * <li>call {@link #build()} to create an {@link AnnotationInfo}.</li>
  * </ol>
  * One builder instance should not be used to create multiple annotations.
+ * <p>
+ * Note that values of <em>all</em> members of given annotation type must be defined
+ * before calling {@code build()}. If they are not, {@code build()} will throw an exception.
```

Comment:
Should we also explicitly specify that defining members that are _not_ present on the annotation type is ignored? Or maybe it should lead to an exception?

Reply:
I assume you mean annotation values with no default values?

Reply:
Done.